src/CMakeLists.txt

```diff
@@ -32,6 +34,14 @@ find_library(LIB_DART_RELEASE
     HINTS "${DART_DIR}/out/ReleaseX64/obj/runtime/bin" "${DART_DIR}/xcodebuild/ReleaseX64/obj/runtime/bin"
 )
 
+add_custom_command(TARGET dart_dll
```

Comment:
Sorry, what is this custom command here to support?

Reply:
Sorry i have forgot todo delete it bevor commit.
I will use this als git Module see https://github.com/DerKleinePunk/HostDartVM
so I test some thing